react/components/molecules/key-value/key-value.js

```diff
@@ -56,8 +58,20 @@ export class KeyValue extends mix(PureComponent).with(IdentifiableMixin) {
         return [styles.value, { color: this.props.valueColor }];
     };
 
+    _borderStyle = () => {
+        switch (this.props.border) {
+            case "none":
+                return { borderBottomWidth: 0 };
+            case "hard":
+                return { borderBottomWidth: 2, borderBottomColor: "#c0c6d1" };
+            case "soft":

```

Comment:
The already defined border bottom wasn't the hard one? Just asking, since that was what I understood

Reply:
Nope, it was the soft one with `{ borderBottomWidth: 1, borderBottomColor: "#e4e8f0" }`